lib/acts_as_tenant/model_extensions.rb

```diff
@@ -103,21 +87,19 @@ def self.included(base)
 
     module ClassMethods
       def acts_as_tenant(tenant = :account, options = {})
-        ActsAsTenant.set_tenant_klass(tenant)
-
         ActsAsTenant.add_global_record_model(self) if options[:has_global_records]
 
         # Create the association
         valid_options = options.slice(:foreign_key, :class_name, :inverse_of, :optional)
-        fkey = valid_options[:foreign_key] || ActsAsTenant.fkey
```

Comment:
so `ActsAsTenant.fkey` is removed but does `#{tenant}_id".to_sym` work the same

Reply:
yes it's basically the same thing, except it's not stored in a global variable anymore for obvious reasons. instead, we just use the `tenant` variable directly in the closure.